dashboards/llm-d/llm-d.json

```diff
@@ -0,0 +1,2288 @@
+{
+  "displayName": "llm-d dashboard",
+  "dashboardFilters": [
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "cluster",
+      "stringValue": "",
+      "templateVariable": "",
+      "valueType": "STRING"
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "location",
+      "stringValue": "",
+      "templateVariable": "",
+      "valueType": "STRING"
+    },
+    {
+      "filterType": "RESOURCE_LABEL",
+      "labelKey": "namespace",
+      "stringValue": "",
+      "templateVariable": "",
+      "valueType": "STRING"
+    },
+    {
+      "filterType": "USER_METADATA_LABEL",
+      "labelKey": "goog-k8s-cluster-name",
+      "stringValue": "",
+      "templateVariable": "",
+      "valueType": "STRING"
+    }
+  ],
+  "description": "This dashboard has charts displaying throughput, latency, cache usage, and other metrics for running llm-d stacks on GKE.",
+  "labels": {},
+  "mosaicLayout": {
+    "columns": 48,
+    "tiles": [
+      {
+        "height": 4,
+        "width": 48,
+        "widget": {
+          "title": "Networking",
+          "id": "",
+          "sectionHeader": {
+            "dividerBelow": false,
+            "subtitle": "To apply the cluster filter to networking metrics, use the `goog-k8s-cluster-name` label in the filter section."
+          }
+        }
+      },
+      {
+        "yPos": 4,
+        "height": 14,
+        "width": 24,
+        "widget": {
+          "title": "Throughput TX Bytes per Pod",
+          "id": "",
+          "xyChart": {
+            "chartOptions": {
+              "displayHorizontal": false,
+              "mode": "COLOR",
+              "showLegend": false
+            },
+            "dataSets": [
+              {
+                "breakdowns": [],
+                "dimensions": [],
+                "legendTemplate": "",
+                "measures": [],
+                "plotType": "LINE",
+                "targetAxis": "Y1",
+                "timeSeriesQuery": {
+                  "outputFullDuration": false,
+                  "prometheusQuery": "sum by (role, pod) (\n1 / (count by (cluster, pod, node, role) ({\"__name__\"=\"vllm:generation_tokens_total\"}))\n* on (node) group_left(zone) (\n    label_replace(\n        sum by (metadata_system_name, zone)(\n            avg_over_time({\"__name__\"=\"compute.googleapis.com/instance/gpu/throughput_tx_bytes\",\"monitored_resource\"=\"gce_instance\"}[${__interval}]) / ${__interval}\n        ),\n        \"node\",\n        \"$1\",\n        \"metadata_system_name\",\n        \"(.+)\"\n    )\n))",
```

Comment:
I'm still not sure about this PromQL. It seems like the intent is for the `count by (cluster, pod, node, role)` part to count pods per node, but it's actually counting the number of distinct labels per pod.

Reply:
Updated with new suggested PromQL.